help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -0,0 +1,141 @@
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL LINE] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Read [Activate profiles and segments to streaming segment export destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-segment-streaming-destinations.html?lang=en) for instructions on activating audience segments to this destination.
+
+### LINE prerequisites {#prerequisites-destination}
+
+Note the following prerequisites in [!DNL LINE], in order to export data from Platform to your [!DNL LINE] account:
+
+#### You need to have a LINE account {#prerequisites-account}
+
+Go to the [!DNL LINE] [signup](https://account.line.biz/signup) page to register and create an account, if you do not have one already.
+
+#### Gather the LINE channel access token (long-lived) from the LINE developer console {#gather-credentials}
+
+* To allow Platform to access [!DNL LINE] resources you will need the *[!DNL Channel access token (long-lived)]* from the desired [!DNL LINE] *Messaging API* channel.
+* Log in with your [!DNL LINE] account to the [[!DNL LINE] Developer console](https://developers.line.biz/console).
+* Next, access the *[!DNL Providers]* list, then select the *[!DNL Provider]* of interest and finally select the *Messaging API* channel to access its settings.
+    * If you are accessing the developer console for the first time please follow the [[!DNL LINE] documentation](https://developers.line.biz/en/docs/messaging-api/getting-started/) to complete the steps required to create a provider.
+* Finally navigate to the ***[!DNL Channel access token]*** section and note down the ***[!DNL Channel access token (long-lived)]*** value required within [Authenticate to destination](#authenticate}) step.
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| `[!DNL Channel access token (long-lived)]` | Your [!DNL LINE Channel access token (long-lived)]. | `aaa2112XSMWqLXR7..........nyilFU=` |
+
+Refer to the [[!DNL LINE] documentation](https://developers.line.biz/en/docs/messaging-api/getting-started/) for guidance.
+
+## Supported identities {#supported-identities}
+
+[!DNL LINE] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
```

Comment:
Minor suggestion for grammar and more context: 

`[!DNL LINE] supports the update and export of identities described in the table below.`